app/validators/blocks/grand_calculated_summary_block_validator.py

```diff
@@ -0,0 +1,100 @@
+from app.validators.blocks.calculation_block_validator import CalculationBlockValidator
+from app.validators.questionnaire_schema import (
+    find_dictionary_duplicates,
+    find_duplicates,
+)
+
+
+class GrandCalculatedSummaryBlockValidator(CalculationBlockValidator):
```

Comment:
This is currently not being used for real validation. You need to update the factory method mapping https://github.com/ONSdigital/eq-questionnaire-validator/blob/main/app/validators/blocks/__init__.py#L17-L25

Reply:
Thank you - fixed 👍